test/rate_limiter_test.ts

```diff
@@ -0,0 +1,252 @@
+import { expect } from '@0x/contracts-test-utils';
```

Comment:
we should add a test for `parseUtils.parseJsonStringForMetaTransactionRateLimitConfigOrThrow`